services/blockchain-indexer/shared/dataService/business/transactionsEstimateFees.js

```diff
@@ -244,6 +246,30 @@ const calcAdditionalFees = async (transaction) => {
 	return additionalFees;
 };
 
+const validateTransactionParams = async transaction => {
+	// Mock optional values if not present before schema validation.
+	if (transaction.module === MODULE.TOKEN
+		&& transaction.command === COMMAND.TRANSFER_CROSS_CHAIN) {
+		if (!('messageFee' in transaction.params)) {
+			transaction.params.messageFee = '10000000';
```

Comment:
would it be better for code consistency to use constants instead of hardcoding the value?